schema/tokite_repositories.schema

```diff
@@ -1,6 +1,7 @@
 create_table "tokite_repositories", force: :cascade do |t|
   t.string   "name",       limit: 200, null: false
   t.string   "url",        limit: 200, null: false
+  t.boolean  "private",                null: false, default: false
```

Comment:
PostgreSQL only supports adding a new column at the last position. ridgepole also warns about it.

> [WARNING] PostgreSQL doesn't support adding a new column except for the last position. tokite_repositories.private will be added to the last.

Reply:
Thanks, I fixed it.